### PR TITLE
[WFCORE-6544] CVE-2023-4061 Management RBAC permission allows unexpected reading of system-properties via resolve-expression

### PR DESCRIPTION
**Pull request by bstansberry**
https://issues.redhat.com/browse/WFCORE-6544

https://access.redhat.com/security/cve/CVE-2023-4061

### CODE REVIEW

**Comment by bstansberry** [comment]
@darranl Please review

File: testsuite/rbac/src/test/java/org/jboss/as/test/integration/mgmt/access/ResolveExpressionTestCase.java [hunk 1]
```diff
@@ -0,0 +1,109 @@
+package org.jboss.as.test.integration.mgmt.access;
```
Comment (yersan) [inline testsuite/rbac/src/test/java/org/jboss/as/test/integration/mgmt/access/ResolveExpressionTestCase.java:1]: We should add a license header later, it is not a hard requirement.

**Review by yersan — APPROVED** [review]

**Comment by bstansberry** [comment]
Thanks @yersan. Right before merging I added a commit to add the license header.

**Review by darranl — COMMENTED** [review]
Great

**Pull request merged**